src/components/TokenSelector/TokenSelector.tsx

```diff
@@ -98,12 +94,66 @@ export default function TokenSelector(props: Props) {
     );
   });
 
+  const sortedFilteredTokens = useMemo(() => {
+    const tokensWithBalance = showBalances
```

Comment:
not critial but we can create 2 arrays in one loop

just 
const tokensWithBalance = []
const tokensWithoutBalance = []
```
if(token.balance?.gt(0) {
 tokensWithBalance.push(token)
} else {
tokensWithoutBalance.push(token)
}


```